### PR TITLE
wiring up share button in side menu

### LINKED ISSUE #837: Implement new back button
As per design in #806 
SVG asset: https://dl.dropboxusercontent.com/u/43783651/Mozilla/back.svg

### LINKED ISSUE #918: Remove publish button, add link to Share in slide out menu
Should use the exact same link

### PR DESCRIPTION
**Pull request by gvn**
Resolves #918 

R? @k88hudson

### CODE REVIEW

File: views/make/index.js [hunk 1]
```diff
@@ -100,6 +100,13 @@ module.exports = view.extend({
             self.$dispatch('openModalPrompt');
         });
 
+        self.$on('sideMenuShareClick', function (event) {
+            document.location =
```
Comment (k88hudson) [inline views/make/index.js]: you don't want `document.location` here, this will cause a page refresh -- require `page` and use `page('/make...');`

**Comment by k88hudson** [comment]
Added a note to the PR

**Comment by gvn** [comment]
R? @k88hudson

**Comment by k88hudson** [comment]
R+

**Pull request merged**